Bug: https://trac.sagemath.org/ticket/30906

From ad702bf59ea7372ce12c37f53a18662cd9baee0b Mon Sep 17 00:00:00 2001
From: Dima Pasechnik <dimpase@gmail.com>
Date: Fri, 13 Nov 2020 12:44:45 +0000
Subject: do the bnfisunit check in Pari, not rely on output

Output format changed in Pari 2.13, so we need to act.
cf https://trac.sagemath.org/ticket/30801#comment:6
---
 build/pkgs/pari/spkg-configure.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/sage/build/pkgs/pari/spkg-configure.m4
+++ b/sage/build/pkgs/pari/spkg-configure.m4
@@ -78,8 +78,8 @@
            sage_spkg_install_pari=yes
         fi
         AC_MSG_CHECKING([whether bnfisunit bug of pari 2.11.3 is fixed])
-        bug_check=`echo "bnf = bnfinit(y^4-y-1); bnfisunit(bnf,-y^3+2*y^2-1)" | $GP -qf 2>> config.log`
-        expected="[[0, 2, Mod(0, 2)]]~"
+        bug_check=`echo "bnf = bnfinit(y^4-y-1); bnfisunit(bnf,-y^3+2*y^2-1)==[[0,2,0]]~" | $GP -qf 2>> config.log`
+        expected="1"
         if test x"$bug_check" = x"$expected"; then
            AC_MSG_RESULT([yes])
         else
